spacy/training/loggers.py

```diff
@@ -133,6 +152,21 @@ def log_step(info: Optional[Dict[str, Any]]):
                     wandb.log({f"loss_{k}": v for k, v in losses.items()})
                 if isinstance(other_scores, dict):
                     wandb.log(other_scores)
+                if model_log_interval and info.get("output_path"):
+                    if info["step"] % model_log_interval == 0 and info["step"] != 0:
+                        log_dir_artifact(
+                            path=info["output_path"],
+                            name="pipleine_" + run.id,
+                            type="checkpoint",
+                            metadata=info,
+                            aliases=[
+                                "epoch {} step {}".format(info["epoch"], info["step"]),
```

Comment:
Another nitpicking comment (sorry), I think we'd typically use f-strings here:
```suggestion
                                f"epoch {info['epoch']} step {info['step']}",
```
(I haven't run this so please double check that I haven't made a typo ;-))